src/main/java/stringCalculator/User.java

```diff
@@ -0,0 +1,27 @@
+package stringCalculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class User {
+
+    public List<String> getUserInput() {
+        Scanner scanner = new Scanner(System.in);
+        String[] userInputTokens;
+        String userInput = scanner.nextLine();
+
+        userInputTokens = userInput.split(" ");
```

Comment:
변수의 `선언`과  `할당`은 동시에 진행해주세요! 🤗

```suggestion
        String userInput = scanner.nextLine();
        String[] userInputTokens = userInput.split(" ");
```